fhir-resource-tooling/src/test/resources/example-cos-credentials.json

```diff
@@ -0,0 +1,6 @@
+{
+  "apikey": "<apikey from COS service credentials>",
```

Comment:
Are we tied to IBM Cloud Object store with this? I would think we would want to use the more generic hmac keys (access key and secret key) instead of IBM Cloud specific apikeys

Reply:
Updated to use HMAC